src/client/Microsoft.Identity.Client/LogLevel.cs

```diff
@@ -9,24 +9,29 @@ namespace Microsoft.Identity.Client
     /// </summary>
     public enum LogLevel
     {
+        /// <summary>
```

Comment:
yes it will
maybe use -1 for heath metrics and not change the others?

Reply:
Are there a negative consequence of using a negative enum value? 
I think the underlying type is a signed int, so this is ok.
`default(LogLevel)` returns 0 (`LogLevel.Error`)  still. Will this cause issues in the code?

Reply:
https://social.msdn.microsoft.com/Forums/vstudio/en-US/805eb2c7-ce76-403d-ad3f-614cab453b58/best-practices-for-negative-enumeration-values?forum=csharpgeneral details 

Looks like Enum.GetValues will return a "strange" ordering of elements, but otherwise seems ok?